src/manipulation.js

```diff
@@ -33,9 +33,27 @@ var
 		option: [ 1, "<select multiple='multiple'>", "</select>" ],
 
 		thead: [ 1, "<table>", "</table>" ],
-		col: [ 2, "<table><colgroup>", "</colgroup></table>" ],
-		tr: [ 2, "<table><tbody>", "</tbody></table>" ],
-		td: [ 3, "<table><tbody><tr>", "</tr></tbody></table>" ],
+
+		// Following wrappers are not fully defined because
+		// their parent elements (except for "table" element) could be omitted
+		// since browser parsers are smart enough to auto-insert them
+		col: [
+
+			2,
+			// Support: Android 2.3
+			// Android browser doesn't auto-insert colgroup element
+			"<table><colgroup>", "</colgroup></table>"
+		],
+
+		tr: [
+			2, // - auto-inserts "tbody" element
```

Comment:
> I'm actually not sure

Yep, me too, hence the question :-). 

> http://contribute.jquery.org/style-guide/js/#comments

Yeah, checked that, you example is also not applicable, since 

> Comments are always preceded by a blank line

So it sounds like, this should look like this - 

```js
// Following wrappers are not fully defined because
// their parent elements (except for "table" element) could be omitted
// since browser parsers are smart enough to auto-insert them

// Support: Android 2.3
// Android browser doesn't auto-insert colgroup element
col: [ 2, "<table><colgroup>", "</colgroup></table>" ],

...

// - auto-inserts "tbody" and "tr" elements
td: [ 3, "<table>", "</table>" ],
```

Which i guess is fine, but less obvious